Frameworks/CoreText/CTRun.mm

```diff
@@ -252,9 +252,18 @@ void _CTRunDraw(CTRunRef run, CGContextRef ctx, CFRange textRange, bool adjustTe
     // Fix this once CTFont and UIFont become bridgable classes.
     id fontColor = [curRun->_attributes objectForKey:(id)kCTForegroundColorAttributeName];
     if (fontColor == nil) {
-        fontColor = [_LazyUIColor blackColor];
+        CFBooleanRef useContextColor =
+            static_cast<CFBooleanRef>([curRun->_attributes objectForKey:(id)kCTForegroundColorFromContextAttributeName]);
+        if (useContextColor && CFBooleanGetValue(useContextColor)) {
+            // Leave the context fill color as is
+        } else {
```

Comment:
Why have an empty true case? Why not just !(useContextColor && CFBooleanGetValue(useContextColor))?